TSPL.docc/ReferenceManual/Attributes.md

```diff
@@ -80,10 +80,12 @@ indicates the macros role:
   These macros add accessors to the stored property they're attached to,
   turning it into a computed property.
 
-- term Conformance macros:
-  Write `conformance` as the first argument to this attribute.
-  The type that implements the macro conforms to the `ConformanceMacro` protocol.
-  These macros add protocol conformance to the type they're attached to.
+- term Extension macros:
+  Write `extension` as the first argument to this attribute.
```

Comment:
The `conformances:` argument comes after `extension`, and it's only necessary if the macro adds extensions with a conformance clause. The `names:` argument comes last, and it's only necessary if the macro adds declarations inside the extension.

Reply:
Added discussion of `conformances:` in d444c690.

Reply:
Added `names:` in 198a4c31 — and fixed that argument's spelling.